recipes/mne-python/construct.yaml

```diff
@@ -155,25 +76,23 @@ specs:
   - libblas =3.9.0=*openblas
   - jupyter =1.1.1
   - jupyterlab =4.4.1
+  - nbclassic =1.3.1
   - ipykernel =6.29.5
   - spyder-kernels =3.0.3
   # TODO: Needs to not require pyqt
   # https://github.com/spyder-ide/spyder/issues/20201
   # spyder =6.1.0
   - darkdetect =0.8.0
   - qdarkstyle =3.2.3
```

Comment:
these are MNE GUI dependencies, probably not needed?
```suggestion
```

Reply:
Ok - removed.